assets/logo/logo-type.svg

```diff
@@ -1,72 +1,27 @@
-		c-1.721,2.562-2.579,6.681-2.579,12.36c0,7.361,0.879,12.6,2.64,15.72c1.759,3.12,4.639,4.681,8.64,4.681
-		c3.84,0,6.619-1.261,8.34-3.78c1.72-2.521,2.58-6.66,2.58-12.421c0-7.278-0.899-12.498-2.699-15.659
-		c-1.801-3.16-4.662-4.74-8.58-4.74C702.701,146.678,699.921,147.959,698.201,150.518z"/>
-	<path fill="#231F20" d="M789.461,181.058c0.399,0.521,1.119,0.78,2.16,0.78c0.799,0,1.839-0.159,3.119-0.479l1.2,0.959l-1.439,8.4
-		c-2.562,1.68-5.48,2.68-8.761,3c-5.039-0.48-8.24-3.48-9.6-9h-1.2c-4.56,4.4-9.239,7.32-14.04,8.76c-6,0-10.601-1.38-13.8-4.14
-		c-3.2-2.761-4.8-6.739-4.8-11.94l0.36-14.16l-0.601-25.2l15.48-1.32l-0.48,35.4c0,5.681,2.4,8.52,7.2,8.52
-		c3.36,0,6.84-1.561,10.44-4.68v-37.92l15.119-1.32l-0.96,41.52C788.86,179.599,789.06,180.539,789.461,181.058z"/>
-	<path fill="#231F20" d="M834.579,178.958l1.32,1.199l-1.561,9.48c-4.08,2.082-8.24,3.36-12.479,3.84
-		c-10.8-1.119-16.2-6.398-16.2-15.84l0.6-9.96l-0.119-20.64h-7.44l-0.6-0.84l0.84-8.76h7.08v-9.24l14.52-4.68l1.08,0.96l-0.36,12.96
-		h14.28l0.48,0.84l-0.721,8.76H821.02l-0.721,26.88c0,2.562,0.48,4.38,1.44,5.46s2.56,1.62,4.8,1.62
-		C829.099,180.999,831.778,180.32,834.579,178.958z"/>
-</g>
-<g>
-	<path fill="#EEC900" d="M190.878,164.796c0-15.434-12.519-27.951-27.951-27.951c-15.438,0-27.95,12.518-27.95,27.951
-		c0,15.446,12.512,27.962,27.95,27.962C178.359,192.758,190.878,180.243,190.878,164.796z"/>
-</g>
-<g>
-	<path fill="#7C4350" d="M162.927,28.211c43.725,0,82.628,20.579,107.626,52.544l-77.669,40.175
-		c-8.53-5.836-18.84-9.264-29.957-9.264c-29.344,0-53.14,23.789-53.14,53.135c0,29.347,23.796,53.141,53.14,53.141
-		c3.77,0,7.455-0.41,11.009-1.159l77.5,52.01c-23.843,20.314-54.735,32.603-88.509,32.603c-75.437,0-136.592-61.153-136.592-136.594
-		C26.335,89.365,87.49,28.211,162.927,28.211z"/>
-</g>
-<g>
-	<path fill="#231F20" d="M190.878,208.742l46.646,28.158c29.294-6.818,7.493-58.36,7.493-58.36l19.299-2.819l-24.069-60.995
-		l-34.288,17.49C230.481,183.761,190.878,208.742,190.878,208.742z M225.031,156.4c0-4.954,4.019-8.967,8.97-8.967
-		c4.955,0,8.969,4.013,8.969,8.967c0,4.95-4.014,8.967-8.969,8.967C229.05,165.367,225.031,161.35,225.031,156.4z"/>
-</g>
-<g>
-	<path fill="#889EA8" d="M263.633,252.253c21.487-24.13,34.542-55.931,34.542-90.78c0-25.376-6.924-49.131-18.971-69.49
-		l-14.813,7.107c0.256,0.402,0.498,0.812,0.738,1.221c10.924,18.441,16.692,39.595,16.692,61.162
-		c0,29.542-10.804,57.907-30.416,79.935c-0.857,0.89-2.099,2.262-2.968,3.104l13.362,9.777c0.342-0.341,0.896-0.985,1.355-1.505
-		l0.006,0.005c0.104-0.116,0.205-0.234,0.309-0.351c0.055-0.06,0.12-0.134,0.167-0.183L263.633,252.253z"/>
-</g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 875 327" width="875" height="327">
+	<style>
+		tspan { white-space:pre }
+		.shp0 { fill: #ffffff } 
+		.shp1 { fill: #7fc6a4 } 
+		.shp2 { fill: #efc900 } 
+		.shp3 { fill: #7b4350 } 
+		.shp4 { fill: #889ea8 } 
+	</style>
+	<g id="Layer">
```

Comment:
Element id attributes should be unique, are we using the id attribute for something specific?